src/main/java/org/candlepin/subscriptions/product/OfferingSyncController.java

```diff
@@ -171,12 +178,35 @@ private SyncResult syncOffering(Offering newState, Optional<Offering> persistedO
       }
     }
 
-    // Existing capacities might need updated if certain parts of the offering was changed.
-    capacityReconciliationController.enqueueReconcileCapacityForOffering(newState.getSku());
+    // Existing capacities need to be updated if certain parts of the offering changed.
+    if (isCapacityImpacted) {
+      capacityReconciliationController.enqueueReconcileCapacityForOffering(newState.getSku());
+      log.info(
+          "SKU {} attribute change(s) impact capacity records, reconciliation scheduled",
+          newState.getSku());
+    } else {
+      log.info("SKU {} attribute change(s) did not impact capacity", newState.getSku());
+    }
 
     return SyncResult.FETCHED_AND_SYNCED;
   }
 
+  private boolean offeringChangeImpactsCapacityRecords(
+      Offering newState, Optional<Offering> persistedOffering) {
+    if (persistedOffering.isEmpty()) {
+      return true;
+    }
+
+    var existingState = persistedOffering.get();
+
+    return !Objects.equals(newState.getCores(), existingState.getCores())
```

Comment:
Seems good. Just verifying SLA and Usage are not part of capacity correct?

Reply:
Correct.

Reply:
Correct